applications/diag/diag-plugins/org.csstudio.diag.pvfields/src/org/csstudio/diag/pvfields/model/EPICSDataProvider.java

```diff
@@ -66,10 +67,17 @@ public void pvChanged(final PVReaderEvent<VType> event)
                 // No error:
                 final String full_name;
 
-                if (name.indexOf("://") > 0)
+                if (name.indexOf("://") > 0) {
                     full_name = name;
-                else
-                    full_name = ConfigurationHelper.defaultDataSourceName() + "://" + name;
+                } else {
+                    DataSource defaultDataSource = PVManager.getDefaultDataSource();
+                    if (defaultDataSource instanceof CompositeDataSource) {
+                        CompositeDataSource cds = ((CompositeDataSource) defaultDataSource);
+                        full_name = cds.getConfiguration().getDefaultDataSource() + cds.getConfiguration().getDelimiter() + name;
```

Comment:
@kasemir I remove utility.pvmanager.ConfigurationHelper, and it's not in diirt.  I tried my best, considering it is 8pm before Thanksgiving :)